spec/services/service_spec.rb

```diff
@@ -0,0 +1,13 @@
+describe "Service" do
```

Comment:
i like that you have a test for your service. i suggest having a test for your Event PORO as well 👍